sdk/eventhubs/azure-messaging-eventhubs/src/consumer_client.cpp

```diff
@@ -55,7 +55,7 @@ namespace Azure { namespace Messaging { namespace EventHubs {
     // Tear down the sessions and then the connections, in that order.
     for (auto& sender : m_receivers)
     {
-      sender.second.Close();
+      sender.second.Close({});
```

Comment:
If I understand correctly, this parameter has a default value, so could/should be written as `Close()`?

Reply:
It could. This change was made before it was defaulted. I didn't bother to change it because ultimately it doesn't matter.

Reply:
I'd recommend to put it simply as `()` rather than `({})` - less questioning, if it matters and why it is written this way, which could take a few minutes to confirm (when you are investigating strange bugs, and don't know where to start, you question everything).

I would think, that there are two overloads of `Close()` - one with a parameter, another one is parameterless. And with syntax your intent is to call the one with a parameter, rather than the parameterless one.